drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/fromfile/DecisionTableAnalyzerFromFileTest.java

```diff
@@ -142,6 +142,24 @@ public void testFile3() throws
     }
 
     @Test
+    public void testCrazyConflictMessages() throws
```

Comment:
I know it was volition but perhaps a better test name would be good?

Reply:
Like "testReallyCrazyConlictMessages()" ?

Ok ok, I'll try to think something that describes this.